src/Components/Web.JS/src/Rendering/EventDelegator.ts

```diff
@@ -170,6 +174,10 @@ class EventInfoStore {
     this.addGlobalListener(info.eventName);
   }
 
+  public get(eventHandlerId: number) {
```

Comment:
```suggestion
  public get(eventHandlerId: number): EventHandlerInfo {
```
like in `public remove(eventHandlerId: number): EventHandlerInfo {`

Reply:
Reasonable suggestion (thanks @campersau!), but this is completely internal code so this change wouldn't really affect anything. We can change it further at any time.

Reply:
Yeah, I think TypeScript's contextual typing will take care of implicitly setting the correct type here.